### PR TITLE
[Lint] Fix Moneo all the lint error

### PR DESCRIPTION
**Pull request by RyoYang**
**Major**

* Add a flake8 config and reset some parameters
* Fix all the lint errors on moneo
* Update to python3 on `src/worker/README`

### CODE REVIEW

File: src/worker/exporters/amd_exporter.py [hunk 1]
```diff
@@ -7,15 +7,16 @@
...
 
+from rsmiBindings import rocmsmi, rsmi_status_t
+from RdcReader import RdcReader
+from rdc_bootstrap import *  # noqa: F403
+
 sys.path.extend([
     '/opt/rocm/libexec/rocm_smi',   # ROCm >=5.2
     '/opt/rocm/rocm_smi/bindings',  # ROCm <5.2
     '/opt/rocm/rdc/python_binding',
 ])
```
Comment (rafsalas19) [inline src/worker/exporters/amd_exporter.py:18]: Is this going to work with the path extensions after the import?

**Comment by RyoYang** [comment]
> 
Thanks for catching this and detailed explanation. Indeed, this way could trigger E402 rule, so I added it into the config and revert the most changes but forgot this one...

File: src/worker/exporters/amd_exporter.py (hunk 1 above)
Comment (RyoYang) [reply src/worker/exporters/amd_exporter.py:14]: Thanks for catching and addressed.

File: src/worker/README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ sudo bash install/nvidia.sh
 # start DCGM daemon
 sudo nv-hostengine &
 # start DCGM exporter (DCGM only supports Python2 currently)
-python2 exporters/nvidia_exporter.py &
+python3 exporters/nvidia_exporter.py &
```
Comment (rafsalas19) [inline src/worker/README.md:13]: good catch

**Review by rafsalas19 — APPROVED** [review]
Hey this looks good. It has taken me some time because of the number of files. One recommendation that I have is that this would have been easier on the team review if the number of modified files per PR was lower. For instance for this PR we could have broken it up into 2-3 PRs. For future PRs let's try to keep the number of files modified below 10.

**Comment by RyoYang** [comment]
> Hey this looks good. It has taken me some time because of the number of files. One recommendation that I have is that this would have been easier on the team review if the number of modified files per PR was lower. For instance for this PR we could have broken it up into 2-3 PRs. For future PRs let's try to keep the number of files modified below 10.

Got it. And will follow that next time.

**Pull request merged**